governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -0,0 +1,411 @@
+  setCurrentProposalPubkey: Dispatch<SetStateAction<string | undefined>>
+}) => {
+  const status = Object.keys(proposal.status)[0]
+
+  const router = useRouter()
+
+  const handleClickIndividualProposal = useCallback(
+    (proposalPubkey: string) => {
+      router.query.proposal = proposalPubkey
+      setCurrentProposalPubkey(proposalPubkey)
+      router.push(
+        {
+          pathname: router.pathname,
+          query: router.query,
+        },
+        undefined,
+        { scroll: false }
+      )
+    },
+    [setCurrentProposalPubkey, router]
+  )
+  return (
+    <div
+      className="my-2 max-h-[58px] cursor-pointer bg-[#1E1B2F] hover:bg-darkGray2"
+      onClick={() =>
+        handleClickIndividualProposal(proposal.publicKey.toBase58())
+      }
+    >
+      <div className="flex justify-between p-4">
+        <div>{proposal.publicKey.toBase58()}</div>
+        <div
+          className={
+            status === 'active'
+              ? 'text-[#E5D8FE]'
+              : status === 'executed'
+              ? 'text-[#15AE6E]'
+              : status === 'cancelled'
+              ? 'text-[#FF5E00]'
+              : status === 'rejected'
+              ? 'text-[#F54562]'
+              : ''
+          }
+        >
+          {status}
+        </div>
+      </div>
+    </div>
+  )
+}
+
+const SignerTag = () => {
+  return (
+    <div className="flex items-center justify-center rounded-full bg-darkGray4 py-1 px-2 text-xs">
+      Signer
+    </div>
+  )
+}
+
+const WritableTag = () => {
+  return (
+    <div className="flex items-center justify-center rounded-full bg-offPurple py-1 px-2 text-xs">
+      Writable
+    </div>
+  )
+}
+
+const Proposal = ({
+  proposal,
+  multisig,
+}: {
+  proposal: TransactionAccount
+  multisig: any
+}) => {
+  const [proposalInstructions, setProposalInstructions] = useState<
+    (PythMultisigInstruction | WormholeMultisigInstruction)[]
+  >([])
+  const { cluster } = useContext(ClusterContext)
+  const anchorWallet = useAnchorWallet()
+  const { squads } = useMultisig(anchorWallet as Wallet)
+
+  useEffect(() => {
+    const fetchProposalInstructions = async () => {
+      const multisigParser = MultisigParser.fromCluster(cluster as PythCluster)
+      if (squads) {
+        const proposalIxs = []
+        for (let i = 1; i <= proposal.instructionIndex; i++) {
+          const instructionPda = getIxPDA(
+            proposal.publicKey,
+            new BN(i),
+            squads.multisigProgramId
+          )[0]
+          const instruction = await squads.getInstruction(instructionPda)
+          const parsedInstruction = multisigParser.parseInstruction({
+            programId: instruction.programId,
+            data: instruction.data as Buffer,
+            keys: instruction.keys as AccountMeta[],
+          })
+          if (
+            parsedInstruction instanceof PythMultisigInstruction ||
+            parsedInstruction instanceof WormholeMultisigInstruction
```

Comment:
You should also display the Unknown instructions

Reply:
Example : https://xc-admin-frontend-r30ihde1s-pyth-web.vercel.app/?tab=proposals&cluster=devnet&proposal=9fsFiFafJLSuLCXtqYxLnp5YTYpy4XLtMkuWATAcnmcT